_i18n/nl.yml

```diff
@@ -242,6 +242,9 @@ faq:
   item18:
     question: Kan ik de IRMA app op Android installeren zonder de Google Play Store?
     answer: Ja, de app is hier te downloaden<span>:</span> <a href="https://f-droid.org/en/packages/org.irmacard.cardemu/">F-droid</a>, of via <a href="https://irma.app/irma.apk" target="_blank">https://irma.app/irma.apk</a>. Als je hem op de laatste manier installeert wordt hij echter niet automatisch geüpdate. 
+  item19:
+    question: Wat is mijn IRMA AppID?
+    answer: De IRMA AppID is een identifier voor je app installatie. En hoe vind je deze AppID? Log aan op <a href="https://privacybydesign.foundation/mijnirma/" target="_blank">MijnIRMA</a>. Na het inloggen staat bovenaan de pagina je AppID. Als je meerdere IRMA apps hebt geinstalleerd, heb je per installatie een eigen AppID. 
```

Comment:
Maybe better: `Als je meerdere IRMA-apps hebt geïnstalleerd, dan heeft iedere installatie een eigen appID.`
(Please pay attention to the ï :) )

In Dutch we once had the convention to say `IRMA-app` everywhere, but I see that goes wrong on this page at more places already.

Reply:
I assume you mean in general when talking about the IRMA-app but not to make it IRMA-app-ID here. Correct?

Reply:
True :)